src/aux-runtime/runtime/AuxLibrary.spec.ts

```diff
@@ -17069,6 +17085,26 @@ describe('AuxLibrary', () => {
                 'data:image/png;base64,AQIDBAU='
             );
         });
+        it('should convert the given value to base64Url string when given an ArrayBuffer', () => {
+            const arrayBuffer = new ArrayBuffer(5);
+            const view = new Uint8Array(arrayBuffer);
+            view[0] = 1;
+            view[1] = 2;
+            view[2] = 3;
+            view[3] = 4;
+            view[4] = 5;
+            expect(library.api.bytes.toBase64Url(arrayBuffer)).toBe(
+                'data:image/png;base64,AQIDBAU='
+            );
+        });
+        it('should throw an error when not given Uint8Array or ArrayBuffer', () => {
+            const invalidInput: any = 'invalid value';
```

Comment:
@TroyceGowdy I just realized that this should be anything but a string, Uint8Array, or ArrayBuffer. `bytes.toBase64Url()` supports strings because it can format a base64 string into a base64 url